index.html

```diff
@@ -101,127 +92,136 @@ <h2 class="header-text">My Recent Works</h2>
-                    </ul>
+                <!-- Fifth -->
+                <div class="work-card">
+                    <h2 class="card-header">Professional Art <br> Printing Data</h2>
+                    <p class="card-description">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                    </p>
+                    <div>
+                        <ul class="card-tags">
+                            <li><button class="tag blur">html</button></li>
+                            <li><button class="tag blur">bootstrap</button></li>
+                            <li><button class="tag blur">Ruby</button></li>
+                        </ul>
+                    </div>
+                    <button class="btn-max-width hide-btn">See Project</button>
+                </div>
+                <!-- sixth -->
+                <div class="work-card">
+                    <h2 class="card-header">Professional Art <br> Printing Data</h2>
+                    <p class="card-description">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                    </p>
+                    <div>
+                        <ul class="card-tags">
+                            <li><button class="tag blur">html</button></li>
+                            <li><button class="tag blur">bootstrap</button></li>
+                            <li><button class="tag blur">Ruby</button></li>
+                        </ul>
+                    </div>
+                    <button class="btn-max-width hide-btn">See Project</button>
                 </div>
-                <button class="btn-max-width">See Project</button>
             </div>
         </div>
     </div>
     </section>
     <section class="about-section">
-        <div>
-            <h1  class="primary-text">about me</h1>
-        </div>
-        <div>
-            <h2 class="description">
-            Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-            </h2>
+        <div class="about-container">
+            <div>
+                <h1  class="primary-text">about me</h1>
+            </div>
+            <div>
+                <h2 class="description">
+                Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </h2>
+            </div>
+            <button class="main-btn no_wrap">Get My Resume</button>
         </div>
-        <button class="main-btn no_wrap">Get My Resume</button>
         <hr class="hl-white">
     </section>
     <section class="skills-section">
```

Comment:
- Please consider adding more spacing between skills for each section. Kindly refer below screenshot 
- Please kindly fix the extra space on the right side of the section.

Figma
<img width="890" alt="Screenshot 2022-12-10 at 10 24 54 PM" src="https://user-images.githubusercontent.com/24832846/206868130-e6a0fa93-8ec7-4724-86b0-edfd7dbd5c31.png">

Your design
<img width="1440" alt="Screenshot 2022-12-10 at 11 11 26 PM" src="https://user-images.githubusercontent.com/24832846/206868148-2bc0a4a4-7b8c-4e67-86e5-a5c9c5823cd2.png">

Reply:
- Added width
- added padding between skills